### PR TITLE
Compute SDI on non-NLM denoised images 

### PR DESCRIPTION
**Pull request by pdedumast**
When `do_nlm_denoised=true`, motion estimation is computed on these images and it generates an SDI. To avoid using an SDI computed with denoised images as reference on the TV, we create a new interface which computes an SDI from non-denoised images using the transforms computed on the denoised ones.

### CODE REVIEW

**Comment by pdedumast** [comment]
Additionally, coming back to the last commit on [the master branch on January 11](https://github.com/Medical-Image-Analysis-Laboratory/mialsuperresolutiontoolkit/pull/125), same problem occurs...

File: pymialsrtk/interfaces/reconstruction.py [hunk 1]
```diff
@@ -448,3 +448,130 @@ def _list_outputs(self):
...
+class MialsrtkSDIComputationOutputSpec(TraitedSpec):
+    """Class used to represent outputs of the
+    MialsrtkSDIComputation interface."""
+
+    output_sdi = File(desc='Output scattered data interpolation image file')
+
+
+class MialsrtkSDIComputation(BaseInterface):
+    """Creates a high-resolution image
+    from a set of low resolution images Todo.
```
Comment (t-sanchez) [inline pymialsrtk/interfaces/reconstruction.py:493]: Why is there a todo here?

File: pymialsrtk/pipelines/anatomical/srr.py [hunk 2]
```diff
@@ -401,6 +401,19 @@ def create_workflow(self):
         srtkImageReconstruction.inputs.sub_ses = sub_ses
         srtkImageReconstruction.inputs.no_reg = self.m_skip_svr
 
+        if self.m_do_nlm_denoising:
```
Comment (t-sanchez) [inline pymialsrtk/pipelines/anatomical/srr.py:404]: Would the code need an explanation of what `m_do_nlm_denoising` means after the change?
i.e. a document clearly somewhere that before, with `m_do_nlm_denoising` the SR reconstruction got the NLM-denoised SDI as input, and that now, it gets the SDI done on the low resolution data.

File: src/MotionEstimation/mialsrtkSDIComputation.cxx [hunk 3]
```diff
@@ -0,0 +1,318 @@
+
+/*=========================================================================
+
+Program: Performs Scattered Data Interpolation from a set of LR images, applying transforms
+Language: C++
+Date: $Date: 2012-28-12 14:00:00 +0100 (Fri, 28 Dec 2012) $
```
Comment (t-sanchez) [inline src/MotionEstimation/mialsrtkSDIComputation.cxx:6]: Update the header to reflect the more recent creation of the file. Mention that it's a stripped down version from `mialsrtkImageReconstruction.cxx` used for convenience.

**Review by t-sanchez — COMMENTED** [review]
A couple of comments on the content, mostly about documentation, clarification of a couple of items.

**Pull request merged**